build.sbt

```diff
@@ -30,14 +30,16 @@ libraryDependencies ++= Seq(
   "com.datastax.cassandra" % "cassandra-driver-core" % cassandra,   //was: 2.1.9
   "org.scalatest"          %% "scalatest"            % "2.2.6"       % "test,it",
   "org.mockito"            % "mockito-core"          % "2.0.34-beta" % "test,it",
-  "ch.qos.logback"         % "logback-classic"       % "1.0.7"       % "test,it",
-  CrossVersion.partialVersion(scalaVersion.value) match {
-    case Some((2, minor)) if minor < 11 =>
-      "org.slf4j"                  % "slf4j-api"     % "1.7.13"
-    case _ =>
-      "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0"
-  }
-)
+  "ch.qos.logback"         % "logback-classic"       % "1.0.7"       % "test,it"
+) ++ (CrossVersion.partialVersion(scalaVersion.value) match {
+  case Some((2, minor)) if minor < 11 => Seq(
+    "org.slf4j"                  % "slf4j-api"     % "1.7.13"
+  )
+  case _ => Seq(
+    "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0",
+    "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.4"
```

Comment:
Since it comes with 2.10 or less, I added this dependency only for scala versions >= 11.